config/default.yaml

```diff
@@ -300,6 +300,9 @@ ecosystem:
     max_size_mb: 0
     max_go_threads: 10000
 
+# environment release information
+release: {}
```

Comment:
Looks like the previous sample was removed.

Reply:
Yeah, we don't need to keep that, default can be empty {}

Reply:
I liked having the previous default, it helped anyone overriding to know what exactly should go in `RELEASE_ENVIRONMENT_VARIABLES`.

Reply:
ok, although if we keep the default values the cookie we will need the override in release.